krabs/krabs/etw.hpp

```diff
@@ -185,6 +197,25 @@ namespace krabs { namespace details {
         return query_trace();
     }
 
+    template <typename T>
+    void trace_manager<T>::set_trace_information(
+        TRACE_INFO_CLASS informationClass,
```

Comment:
let's use snake_case for these three parameter names per the comment in etw.hpp

Reply:
Done.